docs/DeveloperGuide.md

```diff
@@ -293,9 +293,11 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 1.  Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
 2.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
-3.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
-
-*{More to be added}*
+3.  Should be usable by a tech novice who is not familiar with CLI. 
+4.  Should respond within 2 seconds.
+5.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+6.  Simple interface that is easy to navigate.
+7.  Not required to handle finance-related bookkeeping.
```

Comment:
I feel that this line a bit weird? It does make sense though if we have the feature to take note if someone has paid for the upcoming program.

Reply:
oh, so we will include finance-related stuff also? okay no prob i will remove number 7!